Core/NoOpRunner.Core/Dtos/PlayerStateDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿using NoOpRunner.Core.Enums;
+
+namespace NoOpRunner.Core.Dtos
+{
+    public class PlayerStateDto
+    {
+        public PlayerOneState State { get; set; }
+        
+        public bool IsLookingLeft { get; set; }
+        
+        public int XPosition { get; set; }
```

Comment:
CenterPosX is not in center of shape or smtg. its bottom left corner corner of shape

Reply:
It's not center of the shape, it's just the coords from which the offset of the ShapeBlocks is provided. It's no, however, always the left corner of the shape, I think with some of the powerups for example the center pos was two cells below the actual shape. It may have been better to just always use the lowest left ShapeBlock coords as the anchor point, but it's probably not worth changing it at this point, and I guess this version is more flexible.